modules/tensor_mechanics/src/materials/ComputeFiniteStrainElasticStress.C

```diff
@@ -55,12 +50,9 @@ ComputeFiniteStrainElasticStress::computeQpStress()
   // Calculate the stress in the intermediate configuration
   RankTwoTensor intermediate_stress;
 
-  // Check if the elasticity tensor has changed values
-  if (!_is_elasticity_tensor_guaranteed_constant_in_time)
```

Comment:
Do we still need to check for if the elasticity tensor is isotropic?

Reply:
Probably we need the check for isotropic elasticity tensor.  It wasn't there originally, but i'll include it in the current version.

Reply:
Oh, i take that back.  ComputeFiniteStrainElasticStress can only be used for isotropic tensors - there's a check in initialSetup that throws a mooseError otherwise.